### PR TITLE
[Backport][ipa-4-6] Test if WSGI worker process count is set to 4

### PR DESCRIPTION
**Pull request by mrizwan93**
related ticket : https://pagure.io/freeipa/issue/7587

Signed-off-by: Mohammad Rizwan Yusuf <myusuf@redhat.com>
Reviewed-By: Tibor Dudlak <tdudlak@redhat.com>
Reviewed-By: Rob Crittenden <rcritten@redhat.com>
Reviewed-By: Christian Heimes <cheimes@redhat.com>
Reviewed-By: Florence Blanc-Renaud <frenaud@redhat.com>

### CODE REVIEW

**Comment by mrizwan93** [comment]
manual backport of https://github.com/freeipa/freeipa/pull/2246

File: ipatests/test_integration/test_installation.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 import pytest
 from ipalib.constants import DOMAIN_LEVEL_0
+from ipaplatform.constants import constants
```
Comment (Tiboris) [inline ipatests/test_integration/test_installation.py]: Let me try to add line:
```python
from __future__ import absolute_import
```
before your imports.

**Comment by Tiboris** [comment]
Nevermind, just pushed commit with line:
https://github.com/freeipa/freeipa/pull/2296/files#diff-d25674201473f944ed6f4159481c572cR10

**Comment by flo-renaud** [comment]
works for me, ACK.

**Comment by Tiboris** [comment]
ipa-4-6:

* 6760ac910ea5f70c30517bf9e4d77e3064a66085 Test if WSGI worker process count is set to 4

**Pull request closed without merging**